content/200-concepts/100-components/02-prisma-client/055-aggregation-grouping-summarizing.mdx

```diff
@@ -470,6 +471,44 @@ const usersWithCount = await prisma.user.findMany({
 </cmdResult>
 </CodeWithResult>
 
+#### Filter the relation count
+
+This feature is available from version 4.1.0. To use it, you need to add the [preview feature](/concepts/components/preview-features/client-preview-features#enabling-a-prisma-client-preview-feature) `filteredRelationCount`.
+
+```prisma
+generator client {
+  provider        = "prisma-client-js"
+  previewFeatures = ["filteredRelationCount"]
+}
+```
+
+Use `where` to filter the the fields returned by the `_count` output type. You can do this on both [scalar](/concepts/components/prisma-schema/data-model#scalar-fields) and [relation](/concepts/components/prisma-schema/data-model#relation-fields) fields.
```

Comment:
Sorry I forgot to add it in the notion doc, it also works for composite fields.

Also, small typo: `the the`

Reply:
Thanks, I've fixed the typo and added a mention of composite type fields.